app/javascript/components/GuitarGallery.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Title, TitleSizes, Gallery } from '@patternfly/react-core';
+import GuitarInfo from './GuitarInfo';
+
+// Loading the data from the server:
+
+export const GuitarGallery = () => {
+
+  const getGuitars = async () => {
+    try {
+      const response = await axios.get('/guitar_gallery');
+      if(response.data){
+        setGuitars(response.data);
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  const [guitars, setGuitars] = useState([]);
+
+  useEffect(() => {
+    getGuitars();
+  }, []);
+
+  return (
+    <div>
+      <Title headingLevel="h1" size={TitleSizes['4xl']}>
+        Guitars For Sale:
+      </Title>
+      <Gallery 
+      hasGutter
+      minWidths={{
+        md: '100px',
+        lg: '150px',
+        xl: '200px',
+        '2xl': '300px'
+      }}>
+        {guitars.map(({name, url, price, description}) => (
+          <GuitarInfo key={name} name={name} url={url} price={price} description={description}/>
```

Comment:
Can we use `prettier` for the whole project?
you can read more about it at https://dev.to/knowankit/setup-eslint-and-prettier-in-react-app-357b

I think it's better to start with the first PR

Reply:
This is resolved in #13 👍🏼